profile/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Profile page</title>
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
+    <script defer src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="sha384-pprn3073KE6tl6bjs2QrFaJGz5/SUsLqktiwsUTF55Jfv3qYSDhgCecCxMW52nD2" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="../dist/css/index.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.11.3/font/bootstrap-icons.min.css">
+</head>
+<body>
+    <main class="">
+        <nav class="navbar navbar-expand-lg navbar-dark bg-body-tertiary">
+            <div class="container-fluid">
+              <ul class="navbar-nav mt-4">
+                <li class="nav-item dropdown">
+                    <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+                        <img src="/img/perfil.png" class="rounded-circle" height="46" alt="Portrait" loading="lazy"/>
+                        <div class="navbar-nav">
+                          <a class="nav-link active" aria-current="page" href="#">Profile</a>
+                          <a class="nav-link" href="/feed/index.html">Feed</a>
+                        </div>
+                    </div>
+                </li>
+              </ul>
+            </div>
+        </nav>
+
+        <div class="m-auto col-md-8">
```

Comment:
It may add some structure if you place the "sections" in <section> elements. What do you think?